doc/src/explore/devnet.md

```diff
@@ -51,12 +51,13 @@ Now [set up your Sui CLI client and connect to DevNet](../build/cli-client.md#co
 
 ### Request gas tokens
 
-Request gas tokens by posting your address in the [#devnet-faucet](https://discordapp.com/channels/916379725201563759/971488439931392130) Discord channel if you haven't already and wait for them to be issued.
-
-You can find your address by running:
-```shell
-$ sui client active-address
-```
+1. Join [Discord](https://discord.gg/sui) If you haven’t already.
+1. Find your address by running:
```

Comment:
should this be 2. instead of `1.`?

Reply:
> should this be 2. instead of `1.`?

Thankfully, Markdown gets auto-converted to the correct numbering so we don't have to manually reorder. See:
https://github.com/MystenLabs/sui/blob/61c165840b5665843aa79c7390d9036b4a90073e/doc/src/explore/devnet.md